dash-file-upload-app/app.py

```diff
@@ -0,0 +1,93 @@
+import dash
+from dash import dcc, html, Input, Output, State
+from databricks.sdk import WorkspaceClient
+import io, base64
+
+app = dash.Dash(__name__)
+server = app.server
+
+w = WorkspaceClient()
+
+# Adjust to your target volume
+catalog = 'main'
+schema = 'default'
+volume = 'my-volume'
+volume_path = f"/Volumes/{catalog}/{schema}/{volume}"
+volume_folder = 'my-folder'
+volume_folder_path = f"{volume_path}/{volume_folder}"
```

Comment:
For templates, we don't want paths to be hardcoded because it means that the template won't work for everyone who installs the template.

Reply:
Hi Theo, thanks for reviewing!

We can also have the user specify the volume in the sample app UI. That way, the path would not be hard coded, and the template would deploy out of the box.

However, the user would still have to give the service principal permissions to the volume before the actual upload works. We could add a quick guide/link to the docs in the sample app UI to enable that. Let me know if those changes would help.

Otherwise, if we need to wait for UC volumes as app resources we can close the pull request.